src/app/config.module.ts

```diff
@@ -10,12 +10,11 @@ import { FeatureResolverModule } from './shared/feature-resolver/feature-resolve
   imports: [FeatureResolverModule],
   providers: [
     {
-      provide: GRAPHQL_URI,
-      useValue: environment.graphqlUri
-    },
-    {
-      provide: GRAPHQL_BATCH_SIZE,
-      useValue: environment.graphqlUri
+      provide: GRAPHQL_OPTIONS,
+      useValue: {
+        uri: environment.graphqlUri,
+        batchSize: environment.graphqlBatchSize
```

Comment:
bath size can probably just be hardcoded here - I don't expect it to be something that diverges prod vs dev.

Reply:
bath size. Yeah!